### ISSUE TITLE
Spinner not shown

### ISSUE THREAD

**Issue by chaitanya-ramji** [TareqElMasri/TKSubmitTransitionSwift3#3]
Hi,
I'm not using storyboards in my project and have initialised the button via code. This is the hierachy of my file structure:

|_ CollectionViewController
      |_ LoginCell
           |_ LoginButton : TKTransitionButton

However, on calling the function to begin animation, the spinner doesn't show up or animate. I am calling the function from within the LoginCell class. 

```swift
`loginButton.startFinishAnimation(1) {
            let vc: ViewController = ViewController()
            vc.handleGoogleLogin()
        }`

**Comment by heinst** [comment]
Yeah its not showing up for me either, did you ever figure this out? CC @TareqElMasri

**Comment by nadimalam** [comment]
any updates to the missing spinner?
